### ISSUE TITLE
Not working when making a change using emacs

### ISSUE THREAD

**Issue by AbrahamGreenman** [lwblackledge/file-watcher#9]
When making a change to file using emacs, no prompt is made to atom.

**Comment by lwblackledge** [comment]
Hi Charlie, Can you give me a bit more information like OS, Atom version etc?

Thanks!

**Comment by CharGrnmn** [comment]
OS: OS X El Capitan, 10.11.4, 
Atom: 1.7.2
Other editor used: GNU Emacs 23.1.1 

File open in atom. File opened in emacs, and saved. Going back to Atom editor elicits no prompt after save in Emacs. Thank you :)

**Comment by lwblackledge** [comment]
I don't have a Mac so I tried to repro this on an Ubuntu VM with Atom 1.7.2 and Emacs 24.4.
I did try to get Emacs 23.1 installed but I was not able to build it from source.

I don't see the error with Emacs 24 at first try.
I'll take another stab at Emacs 23 in case it is version specific, but is there a way for you to try with Emacs 24? I admit it does not look as easy to get running on a Mac as it was on Ubuntu...
